OPT_mission.Tanoa/training/functions/fnc_initmyposdialog.sqf

```diff
@@ -0,0 +1,46 @@
+/**
+* Author: James
+* initialize flag dialog with coordinates of CSAT and NATO flag positions
+*
+* Arguments:
+* None
+*
+* Return Value:
+* None
+*
+* Example:
+* [] call fnc_initMyPosDialog.sqf;
+*
+*/
+#include "script_component.hpp"
+
+#define MYPOS_IDC 1412
+
+params [["_dialog", displayNull, [displayNull], 1]];
+
+disableSerialization;
+
+if (_dialog isEqualTo displayNull) exitWith {};
+
+private _edit = _dialog displayCtrl MYPOS_IDC;
+
+// display coordinates of flagpoles nearby
+
+private _lineBreak = toString [10];
+private _dialogText = "";
+
+// get location string from something nearby
+private _location = "";
+{ 
+	scopeName "locationsearch";
+	if (text _x != "") then
+	{
+		_location = text _x;
+		breakOut "locationsearch";
+	};
```

Comment:
Wozu soll denn dieses Konstrukt mit dem `breakOut` gut sein?

Reply:
Die Suche nach einem nahe gelegenen Standort (mit Namen) kann nach dem ersten Treffer abgebrochen werden. Ein normaler break hätte es auch getan - aber den scheint es in ArmA nicht zu geben.

Reply:
Aber so wie der Code hier steht, sollte der Code doch nur aus dem `if` raus springen und nicht aus dem ganzen Loop, oder? :thinking: 

Und das `break` von Arma heißt `exitWith` ;)

Reply:
Doch er sollte ganz raus. Aber mit `exitWith` ist es nochmal eine ganze Ecke lesbarer - Danke für den Tipp!